### PR TITLE
samples: zigbee: Rename cdc_acm_uart node

### PR DESCRIPTION
**Pull request by sebastiandraus**
Renamed cdc_acm_uart0 node to cdc_acm_uart to accomodate to changes in nrf52840 dongle board - cdc_acm_uart node is added by default and is used as backend by logger/console and shell.

### CODE REVIEW

File: samples/zigbee/ncp/boards/nrf52840dongle_nrf52840_usb.overlay [hunk 1]
```diff
@@ -5,14 +5,14 @@
  */
 
 &zephyr_udc0 {
-	cdc_acm_uart0: cdc_acm_uart0 {
+	cdc_acm_uart: cdc_acm_uart {
```
[the remark this answers is not in the document]
Comment (sebastiandraus) [reply samples/zigbee/ncp/boards/nrf52840dongle_nrf52840_usb.overlay:8]: That's my mistake, no additional cdc_acm_uart is required as dongle has already cdc_acm_uart defined at board level. I'll remove this

File: samples/zigbee/shell/app_usb.overlay [hunk 2]
```diff
@@ -5,13 +5,13 @@
  */
 
  &zephyr_udc0 {
-	cdc_acm_uart0: cdc_acm_uart0 {
+	cdc_acm_uart: cdc_acm_uart {
```
[the remark this answers is not in the document]
Comment (jfischer-no) [reply samples/zigbee/shell/app_usb.overlay:8]: > Can we keep it here this way?

okay.

**Comment by sebastiandraus** [comment]
@jfischer-no , branch updated

**Comment by sebastiandraus** [comment]
Just one empty line removed to look pretty

**Pull request closed without merging**